evidence.go

```diff
@@ -200,15 +202,28 @@ func (e *CcaEvidence) Verify(iak crypto.PublicKey, rpk crypto.PublicKey) error {
 		return fmt.Errorf("missing CCA realm Token")
 	}
 
+	//	var rpk crypto.PublicKey
+
+	// extract RAK from the realm token
+	rawRAK, err := e.CcaRealmClaims.GetPubKey()
+	if err != nil {
+		return fmt.Errorf("extracting RAK from the realm token: %w", err)
+	}
+
+	rak, err := ecdsaPKeyFromRaw(rawRAK)
```

Comment:
```suggestion
	rak, err := ecdsaPublicKeyFromRaw(rawRAK)
```

Reply:
Also please do not encode Algorithm in the name of function, as soon we may get another algorithm, we do not want to revisit here again and again!
I would simply like to call it:
getPublicKeyFromRaw

Reply:
the function returns an ecdsa.PublicKey so the name is rather fitting